common/src/main/java/io/netty/util/concurrent/DefaultRejectTaskHandler.java

```diff
@@ -0,0 +1,18 @@
+package io.netty.util.concurrent;
+
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+/**
+ * The default implementation just logs the rejection of a rejected task. 
+ * 
+ */
+public class DefaultRejectTaskHandler implements RejectTaskHandler {
+	private static final InternalLogger rejectedExecutionLogger =
+           InternalLoggerFactory.getInstance(DefaultPromise.class.getName() + ".rejectedExecution");
+
+	@Override
+	public void rejectTask(String reason, Runnable task, Throwable t) {
```

Comment:
* Please rename to `taskRejected()`
* I don't think we need a `reason` parameter here. The `Throwable` already has a message property.
* Please rename `t` to `cause`.